wordpress-mobile/WordPress-iOS
# Reader Improvements: Add "Discover more topics" button to Followed Topics screen

Fixes #15806

## Description
-  Added a "Discover more topics" button to the Followed Topics screen (under the settings icon on Reader), so that users can re-visit the topic picker screen.

## Notes

I noticed that in WPiOS 16.7, if you entered a tag with multiple words, the tag would show up as hyphenated in the list of Followed Topics (i.e. "social media" is displayed as "social-media"). 

I created a separate issue: https://github.com/wordpress-mobile/WordPress-iOS/issues/15985

## How to test
1. Go to the Reader tab
2. Tap the settings icon > tap on "Discover more topics"
3. ✅ The topics picker screen should be presented
4. Tap any topic > tap "Done"
5. ✅ The new topic should show up in the list of Followed Topics
6. Tap on "Discover more topics" again
7. ✅  The topic you added in step 4 should not be included in the list of available topics to follow

## Screenshots
Followed topics screen | Follow topics screen 
--- | ---
![Simulator Screen Shot - iPhone 12 Pro - 2021-02-26 at 16 30 07](https://user-images.githubusercontent.com/6711616/109269644-20ec3580-7850-11eb-9a41-2fb4ddfd11ce.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-02-26 at 16 30 26](https://user-images.githubusercontent.com/6711616/109269647-22b5f900-7850-11eb-970f-e8a0c30eb25d.png)

## PR submission checklist
